### DISCUSSION TITLE
error code -32603

### DISCUSSION THREAD

**Discussion by brightJoo** [community/community#51554] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I use Intellij, Windows

I used before in my desktop.
Sign in failed. Reason: Request signInInitiate failed with message: self signed certificate in certificate chain, request id: 7, error code: -32603

**Comment by Akash1134** [comment]
Hi @brightJoo , please check this [discussion](https://github.com/orgs/community/discussions/41505) and let us know if that helps. Thanks!

**Comment by 355125772** [comment]
I have same problem:
Sign in failed. Reason: Request signInInitiate failed with message: EditorFetcher request failed: Error processing request: null: undefined, request id: 35, error code: -32603
